.github/workflows/uml.yml

```diff
@@ -0,0 +1,49 @@
+name: Update the UML Diagrams
+
+on:
+  pull_request:
+    branches: [main]
+    # paths:
```

Comment:
Remove commented out code?

Reply:
Yes, at end since the files that changed wouldn't trigger this action